dev-python/pytest-qt/metadata.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+	<maintainer type="person">
+		<email>leycec@gmail.com</email>
+		<name>leycec</name>
+	</maintainer>
```

Comment:
Maybe I missed something, but who is this person? Do you have their permission to be added here? We'd like them to ack this.

Reply:
@leycec of the [raiagent overlay](https://github.com/leycec/raiagent). The ebuild here is heavily based on the one found in the raiagent overlay, that's why they are still in the metadata file. I figured it would be impolite to just remove them from the metadata file.

@leycec do you want to be removed from pytest-qt's metadata file here, or would you like to keep it like this?

Reply:
When we push it to ::gentoo, anyone listed here will become the package maintainer and gets assigned bugs, etc is expected to keep answering to the bugs, bump requests and other ebuild maintenance routine.

Ie if you haven't talked to him about this at all, remove him from here. He can be added later with proper consent, or he can start making PRs and add himself.